reports/Virginia.md

```diff
@@ -167,3 +167,13 @@ id: va-richmond-7
 
 * https://twitter.com/aeringle/status/1274898298923606017
 * https://www.reddit.com/r/2020PoliceBrutality/comments/hdycck/more_video_from_richmond_last_night_police_spray/
+
+### Police escale against protestors outside city hall | June 23rd
```

Comment:
Got a bunch of typos in here. escale. cite. 

Can you also include more of the links from the twitter thread as well as an ID and tags.

Reply:
Thanks for that catch. I've implemented those changes.